internal/bminventory/inventory.go

```diff
@@ -508,8 +518,10 @@ func (b *bareMetalInventory) RegisterClusterInternal(
 			Href:                         swag.String(url.String()),
 			Kind:                         swag.String(models.ClusterKindCluster),
 			APIVip:                       params.NewClusterParams.APIVip,
```

Comment:
worth adding some comment about deprecating the singular variants for later on? maybe in the params struct?

Reply:
https://github.com/openshift/assisted-service/pull/4668